dpe/src/dpe_instance.rs

```diff
@@ -871,4 +898,28 @@ pub mod tests {
         // validation fails on a graph with multiple connected components
         assert_eq!(dpe.validate_context_tree(0), false);
     }
+
+    #[test]
+    fn test_new_auto_init() {
```

Comment:
can you add a test to ensure `initialize_context` fails if new_auto_init was used.

Reply:
Done